src/react/actions/user.js

```diff
@@ -144,6 +144,21 @@ export function deleteAccessKey(accessKey, userName) {
   };
 }
 
+export function updateAccessKey(accessKey, status, userName) {
+  return (dispatch, getState) => {
+    const { iamClient } = getClients(getState());
+    dispatch(networkStart('Updating user access key'));
+    iamClient
+      .updateAccessKey(accessKey, status, userName)
+      .then(() => {
+        dispatch(listAccessKeys(userName));
+      })
+      .catch(error => dispatch(handleClientError(error)))
+      .catch(error => dispatch(handleApiError(error, 'byModal')))
+      .finally(() => dispatch(networkEnd()));
+  };
+}
+
```

Comment:
This code is never used, it can be removed

Reply:
Code deleted -> done